conf/app.routes

```diff
@@ -1,4 +1,4 @@
-GET   /:UTR/:TAX_YEAR/ats-data                controllers.ATSDataController.getATSData(UTR: String, TAX_YEAR: Int)
-GET   /:NINO/:TAX_YEAR/paye-ats-data          controllers.ATSPAYEDataController.getATSData(NINO: String, TAX_YEAR: Int)
-GET   /:UTR/has_summary_for_previous_period   controllers.ATSDataController.hasAts(UTR: String)
-GET   /:UTR/ats-list                          controllers.ATSDataController.getATSList(UTR: String)
+GET   /:UTR/:TAX_YEAR/ats-data                @controllers.ATSDataController.getATSData(UTR: String, TAX_YEAR: Int)
+GET   /:NINO/:TAX_YEAR/paye-ats-data          @controllers.ATSPAYEDataController.getATSData(NINO: String, TAX_YEAR: Int)
```

Comment:
can we please use InjectedRoutesGenerator instead?

Reply:
Yeah as part of bootstrap